nni/retiarii/nn/pytorch/api.py

```diff
@@ -377,8 +365,14 @@ def _valuechoice_codegen(*, _internal: bool = False):
                 print(binary_r_template.format(op=op, opt=opt, sym=sym) + '\n')
 
 
-def _valuechoice_staticmethod_helper(orig_func):
-    orig_func.__doc__ += """
+_func = TypeVar('_func')
+_cand = TypeVar('_cand')
+_value = TypeVar('_value')
```

Comment:
when should we use TypeVar? for example, '_func', can I use 'Callable' instead?

Reply:
TypeVar is useful in generics.

https://docs.python.org/3/library/typing.html#typing.TypeVar